defaults.sh

```diff
@@ -50,10 +50,10 @@
 DEF_WIN_ICU_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/icu/icu4c-68_2-Win64-MSVC2019.zip"
 
 # The download URL for the Windows LLVM/Clang binaries
-DEF_WIN_LLVM_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/llvm/libclang-release_80-based-windows-vs2017_32.7z"
+DEF_WIN_LLVM_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/llvm/libclang-release_110-based-windows-vs2019_64.7z"
```

Comment:
Perhaps we should make a clean break here and move the artifacts from "win32-binary-artifacts" to "win64-binary-artifacts"?

Reply:
I thought so too, but there is nothing like win64. The API is still called win32, no matter of the word size. Obviously we are not alone who did not realize, so there are some artifacts with win64 in filename IIRC.

Reply:
The API is no longer called win32, they renamed it to "Windows API" at some point. Even though the term win32 seems to still be there in e.g. the URL: https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-api-list. It's not a big deal, but it just feels wrong to have 64 binaries in a directory with "32" in the name.

Reply:
OK, let's change it to just `win` next time we touch this? (I'd like to get this off the table finally for this round)